inputFiles/poromechanics/PoroElastic_staircase_co2_3d_fim_new.xml

```diff
@@ -0,0 +1,152 @@
+      timeIntegrationOption="QuasiStatic"
+      logLevel="1"
+      discretization="FE1"
+      targetRegions="{ channel, barrier  }"/>
+
+    <CompositionalMultiphaseFVM
+      name="twoPhaseFlow"
+      logLevel="1"
+      discretization="fluidTPFA"
+      targetRelativePressureChangeInTimeStep="1"
+      targetPhaseVolFractionChangeInTimeStep="1"
+      targetRegions="{ channel  }"
+      temperature="300"
+      useMass="1"/>
+
+    <CompositionalMultiphaseWell
+      name="compositionalMultiphaseWell"
+      logLevel="1"
+      targetRegions="{ wellRegion1, wellRegion2 }"
+      maxCompFractionChange="0.2"
+      useMass="1">
+      <WellControls
+        name="wellControls1"
+        logLevel="1"  
+        type="producer"
+        control="BHP"
+        referenceElevation="62.5"
+        targetBHP="9e6"
+        targetPhaseRateTableName="producerPhaseRateTable"
+        targetPhaseName="water"/>
+      <WellControls
+        name="wellControls2"
+        logLevel="1"      
+        type="injector"
+        control="totalVolRate"
+        referenceElevation="250"
+        targetBHP="1.5e7"
+        targetTotalRateTableName="injectorTotalRateTable"
+        useSurfaceConditions="1"
+        surfacePressure="101325"
+        surfaceTemperature="288.71"
+        injectionTemperature="355"
+        injectionStream="{ 1.0, 0.0 }"/>
+    </CompositionalMultiphaseWell>
+    
+  </Solvers>
+
+  <Events
+    minTime="-1e11"  
+    maxTime="1e7">
+    
+    <PeriodicEvent
+      name="outputs"
+      timeFrequency="1e6"
+      target="/Outputs/vtkOutput"/>
+
+    <SoloEvent
+      name="poroelasticityEquilibrationStep"
+      targetTime="-1e11"
+      beginTime="-1e11"
+      target="/Tasks/poroelasticityEquilibrationStep"/>
+
+    <PeriodicEvent
+      name="solverApplications1"
+      endTime="1e3"
+      target="/Solvers/reservoirSystem"/>
+    <PeriodicEvent
+      name="solverApplications2"
+      beginTime="1e3"
+      maxEventDt="1e6"
+      target="/Solvers/reservoirSystem"/>
+
+    <PeriodicEvent
+      name="linearElasticityStatistics"
+      timeFrequency="1e6"
+      target="/Tasks/linearElasticityStatistics"/>
+    <PeriodicEvent
+      name="twoPhaseFlowStatistics"
+      timeFrequency="1e6"
+      target="/Tasks/twoPhaseFlowStatistics"/>
+    
+    <PeriodicEvent
+      name="restarts"
+      timeFrequency="5e6"
+      target="/Outputs/restartOutput"/>
+  </Events>
+
+  <Outputs>
+    <VTK
+      name="vtkOutput"
+      fieldNames="{ initialPressure, skeletonChannel_density }"/>
+    <Restart
+      name="restartOutput"/>
+  </Outputs>
+
+  <Tasks>
+    
+    <SolidMechanicsStatistics
+      name="linearElasticityStatistics"
+      solidSolverName="linearElasticity"
+      logLevel="1"/>
+    <CompositionalMultiphaseStatistics
+      name="twoPhaseFlowStatistics"
+      flowSolverName="twoPhaseFlow"
+      logLevel="1"
+      computeCFLNumbers="1"
+      computeRegionStatistics="1"/>
+
+    <MechanicsInitialization 
```

Comment:
this is how initialization can be defined now

Reply:
I'm a way out of my domain with mechanics and the paradigm for what gets tagged as an event versus a bool field on object is not entirely clear ... but generalizing a bit... would there be a ReservoirInitialization Task.... ??  Whenever/if multiphase initialization is implemented there will be a whole suite of initialization options. What leads the user to think I need a MechanicsInit Task and not a ReservoirInit task. ...